src/annotator/test/highlighter-test.js

```diff
@@ -333,6 +337,128 @@ describe('annotator/highlighter', () => {
     });
   });
 
+  function createHighlights(root) {
```

Comment:
It would be helpful to add a quick comment here about what the results of this function are.